app/lib/lib/validation/messages.js

```diff
@@ -0,0 +1,56 @@
+import { Match, check } from 'meteor/check';
```

Comment:
I've introduced this file in this directory, but I'm not sure this is the best place for it. Anybody has an idea to share about this?